Modules/FV0/src/DigitQcTask.cxx

```diff
@@ -95,11 +95,11 @@ void DigitQcTask::initialize(o2::framework::InitContext& /*ctx*/)
   mMapChTrgNames.insert({ o2::fv0::ChannelData::kIsEventInTVDC, "IsEventInTVDC" });
   mMapChTrgNames.insert({ o2::fv0::ChannelData::kIsTimeInfoLost, "IsTimeInfoLost" });
 
-  mMapDigitTrgNames.insert({ o2::fit::Triggers::bitA, "OrA" });
-  mMapDigitTrgNames.insert({ o2::fit::Triggers::bitAOut, "OrC" });
-  mMapDigitTrgNames.insert({ o2::fit::Triggers::bitAIn, "Vertex" });
-  mMapDigitTrgNames.insert({ o2::fit::Triggers::bitTrgCharge, "Central" });
-  mMapDigitTrgNames.insert({ o2::fit::Triggers::bitTrgNchan, "SemiCentral" });
+  mMapDigitTrgNames.insert({ o2::fv0::Triggers::bitA, "OrA" });
+  mMapDigitTrgNames.insert({ o2::fv0::Triggers::bitAOut, "OrAOut" });
+  mMapDigitTrgNames.insert({ o2::fv0::Triggers::bitTrgNchan, "TrgNChan" });
+  mMapDigitTrgNames.insert({ o2::fv0::Triggers::bitTrgCharge, "TrgCharge" });
+  mMapDigitTrgNames.insert({ o2::fv0::Triggers::bitAIn, "OrAIn" });
```

Comment:
@afurs Did you change the namespace from `o2::fit::Triggers` to `o2::fv0` on purpose here? In O2 the namespace used by `Triggers` class is `o2::fit`. All the others are aliases used temporarily for compatibility (until the detector-namespaced code can be refactored in all the packages that use it outside O2). This is detail related to future code cleanup - and it should not put merging of this PR on hold. Anyway, I'd like to know if there is any reason on your side.

Reply:
@mslupeck ok, I changed it